Cargo.toml

```diff
@@ -41,7 +41,7 @@ url = { version = "2.3.1", default-features = false, features = ["serde"] }
 xz2 = { version = "0.1.7", default-features = false, features = ["static", "tokio"] }
 plist = { version = "1.7.0", default-features = false, features = [ "serde" ]}
 dirs = { version = "5.0.0", default-features = false }
-typetag = { version = "0.2.3", default-features = false }
+typetag = { git = "https://github.com/noamraph/typetag.git", branch = "ignore-repeating-tag", default-features = false }
```

Comment:
Have you submitted the PR upstream to see what kind of feedback they have?

I'm roughly on board with the general idea (every step having its action name recorded as a member of the JSON object as well as the key of the JSON object itself).

However, I am not comfortable depending on a git dependency for a few reasons (not least of which because that means we can't publish to crates.io anymore), so it'd need to be deemed acceptable by upstream and then merged and released before we could make use of that here.

Reply:
Oh, it looks like it was already accepted and merged upstream: https://github.com/dtolnay/typetag/pull/82

If you update the dependency to v0.2.17, I think this is not an issue, then!

Reply:
Sure, that was my intention, it's much better not to depend on a specific github repository.
I updated the version in the latest commit.